etc/config.sample.toml

```diff
@@ -167,6 +167,12 @@
   # increase in cache size may lead to an increase in heap usage.
   series-id-set-cache-size = 100
 
+  # The size of tar buffer window size in bytes while running tar
+  # streaming operations such as renaming and copying tar files during backups.
+  # The default value is 1MB. This should only change if backups are having performance issues
+  # and you understand that this value is a heuristic that may need to be tweaked.
+  # tar-stream-buffer-size = 1048576
```

Comment:
It looks like the tar buffer size is only used in the backup code paths. If this is correct, I wonder if `backup-stream-buffer-size` would be a better name.